src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -99,7 +98,7 @@ object WorkflowDescriptor {
   }
 
   def apply(id: WorkflowId, sourceFiles: WorkflowSourceFiles, conf: Config): WorkflowDescriptor = {
-    validateWorkflowDescriptor(id, sourceFiles, CromwellServer.backend, conf) match {
+    validateWorkflowDescriptor(id, sourceFiles, CromwellBackend.backend(), conf) match {
```

Comment:
They're thoughtcrime except when it is thoughtcrime to *not* do it (aka intellij reminded me)